pom.xml

```diff
@@ -201,10 +201,9 @@
         </dependency>
 
         <dependency>
-            <groupId>io.jenkins</groupId>
-            <artifactId>configuration-as-code</artifactId>
+            <groupId>io.jenkins.configuration-as-code</groupId>
+            <artifactId>test-harness</artifactId>
             <version>${configuration-as-code.version}</version>
-            <classifier>tests</classifier>
```

Comment:
why should it be hard dep?

Reply:
I just removed the classifier, this is still a test dependency.